packages/vue/src/components/confirm-sign-in.vue

```diff
@@ -11,9 +11,11 @@
           style="flex-direction: column"
           :disabled="actorState.matches('confirmSignIn.pending')"
         >
-          <base-heading :level="3" class="amplify-heading">
-            {{ confirmSignInHeading }}
-          </base-heading>
```

Comment:
I added in the change so we are no longer showing the header for sign in / sign up. I left them for the other pages though, since we don't have tabs on them like confirm sign up, etc.

Reply:
Scratch that, I'm adding back the header for now, since it breaks the tests. I'll have that in a different PR.